recipes/pyobjcryst/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "pyobjcryst" %}
+{% set version = "2.0.2" %}
+{% set sha256 = "b48b1a66973dc42d8d15c5ddcf4989cb4160cab220c3a3c60664af46926092d4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - SConstruct.patch
+
+build:
+  number: 0
+  skip: True  # [win or py2k or osx]
```

Comment:
This should build on `osx`, why skip there?

Reply:
OSX build is failing on travis I thought we could debug it in the feedstock though